### ISSUE TITLE
Is the documentation in Readme going to be moved to the appium 2 docs interface?

### ISSUE THREAD

**Issue by ollie-iterators** [appium/appium-espresso-driver#842]
I am wondering if the information in Readme is going to be moved to the appium 2 docs interface like the Readme information in the appium-xcuitest-driver repository in the PR: https://github.com/appium/appium-xcuitest-driver/pull/1475.

**Comment by jlipps** [comment]
Yes, that's scheduled work for this driver. It has not yet begun though (I'm probably going to start work on the uiautomator2 driver first). So if you want to tackle it, I'm happy to help review!

**Comment by eglitise** [comment]
Documentation is now available at https://appium.github.io/appium-espresso-driver/latest/
